lib/settings/allowed_phone_numbers.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+module.exports = (config, Settings, log) => {
+
+  class AllowedPhoneNumbers extends Settings {
+    constructor(phoneNumbers) {
+      super('allowedPhoneNumbers')
+      this.setAll(phoneNumbers)
+    }
+
+    isAllowed(phoneNumber) {
+      return phoneNumber in this.phoneNumbers
+    }
+
+    setAll(phoneNumbers) {
+      this.phoneNumbers = {}
```

Comment:
I wasn't sure if a JavaScript Set is allowed in the customs-server, seems like it'd be a natural fit.

Reply:
I say do it, good shout!